src/index.mako.html

```diff
@@ -726,22 +726,34 @@
         var cacheAdd = '${version}' != '' ? '/' + '${version}' : '';
         var pathname = location.pathname.replace(/(index|mobile|embed)\.html$/g, '');
 
+        function getParam(name){
+          return decodeURIComponent((new RegExp('[?|&]' + name + '=' + '([^&;]+?)(&|#|;|$)').exec(location.search)||[,""])[1].replace(/\+/g, '%20'))||null;
+        }
+
+        var my_api_url = getParam('api_url');
+
+        if (my_api_url){
+          api_url = '//mf-chsdi3.dev.bgdi.ch/' + my_api_url;
```

Comment:
We should think about integration, or other environments as well. So I would rather have here the domain of the ${api_url} variables per default. Then optionally, we can add a 2nd paramter where we can specify the environment. So we would have something like:

```
var my_api_env = getParam('api_env');
api_url = domain_from_url(${api_rul);
if (my_api_env) {
  api_url = '//mf-csdhi3.' + my_api_env + 'bgdi.ch/' + my_api_url;
}